go/exocom-mock/exocom-mock.go

```diff
@@ -38,23 +38,31 @@ func (exoCom *ExoComMock) Listen(port int) error {
 	return exoCom.server.ListenAndServe()
 }
 
-// websocketHandler is called when the given socket connects
```

Comment:
This function just moved to the bottom of the file. Keeping all the public functions up top and putting the private ones at the end